.env.sample

```diff
@@ -242,3 +242,5 @@ AP_USERS_ORGANISATION_ID = ""
 CASSANDRA_HOST = ""
 CASSANDRA_PORT = ""
 CASSANDRA_DB = ""
+
+CLOUD_PATH = ""
```

Comment:
@rakeshSgr - No need to provide folder via env. Use existing uploads folder

Reply:
resolved